egov/egov-collectionweb/src/main/java/org/egov/collection/web/actions/receipts/ChequeRemittanceAction.java

```diff
@@ -205,12 +206,29 @@ public String listData() {
             fundQuery.setString("accountNumberId", accountNumberId);
             List<String> fundCodeList = fundQuery.list();
             final String fundCode = fundCodeList != null && !fundCodeList.isEmpty() ? fundCodeList.get(0).toString() : null;
+            Long dateInLongFromDate=null;
+            Long dateInLongToDate=null;
 
+           
             final CFinancialYear financialYear = financialYearDAO.getFinancialYearById(finYearId);
+            DateFormat formatter = new SimpleDateFormat("dd/MM/yyyy");
+            Date dateFrom;
+            Date toDates;
+            try {
+                dateFrom = formatter.parse(financialYear.getStartingDate().toString());
+                toDates = formatter.parse(financialYear.getStartingDate().toString());
+                dateInLongFromDate = dateFrom.getTime();
+                dateInLongToDate = toDates.getTime();
+
+
+            } catch (ParseException e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            }
             receiptBeanList = remittanceService.findChequeRemittanceDetailsForServiceAndFund("",
                     StringUtils.join(serviceCodeList, ","), fundCode,
-                    fromDate == null ? financialYear.getStartingDate() : fromDate,
-                    toDate == null ? financialYear.getEndingDate() : toDate);
+                    dateInLongFromDate,
```

Comment:
If from date is null then only use the from date from financial year else use the from date itself